app/response_processor.py

```diff
@@ -122,6 +122,7 @@ def _send_receipt(self, decrypted, xml):
 
             response = session.post(endpoint, data=xml, headers=headers, verify=False, auth=auth)
             self.logger = self.logger.bind(status=response.status_code)
+            self.logger.info("Request details", data=xml, header=headers)
```

Comment:
What info is contained in the xml and the headers?  And how big are they?  We don't want huge globs of data in the logs if we can avoid it.  I'm worried about possibly putting something sensitive in the logs

Reply:
I don't think they'll be anything sensitive in the receipt, its just the respondent id and the tx_id. Though I agree we shouldn't log all the xml and headers, just the useful bits

Reply:
What about the request url - is that logged anywhere, as that would probably be quite valuable.

Reply:
That is already bound to the logger

Reply:
If the request url and tx_id are already logged then I'm not sure this PR is needed.